### PR TITLE
Slm 340 update gov.uk header on public service

### PR DESCRIPTION
**Pull request by simonnebMOJ**
I have had a go at updating the Header for the new logo.

### CODE REVIEW

File: server/views/partials/header.njk [hunk 1]
```diff
@@ -1,45 +1,39 @@
...
-            </a>
-          </li>
-
-          <li class="hmpps-header__navigation__item">
-            <a data-qa="signOut" class="hmpps-header__link" href="/sign-out">Sign out</a>
-          </li>
-        {% endif %}
-
+    <div class="govuk-header__content">
+      <a href="/" class="govuk-header__link govuk-header__service-name">
```
Comment (dhirajshettymoj) [inline server/views/partials/header.njk:20]: can we add data-qa="header-text" to this tag as a fix for a failing integration test.

Comment (dhirajshettymoj) [inline server/views/partials/header.njk:20]: can we add data-qa="header-text" to this tag as a fix for a failing integration test.

Comment (simonnebMOJ) [reply server/views/partials/header.njk]: Thanks - updated. Integration tests now passing.

**Review by dhirajshettymoj — APPROVED** [review]
LGTM

**Pull request closed without merging**